lib/repositories/genre_repository.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+class GenreRepository {
+  Future<List<String>> getGenreNamesById(List<num> ids) async {
+    final String response =
+        await rootBundle.loadString('assets/mock_data/genres.json');
+    List<dynamic> genres = json.decode(response)['genres'];
+    return genres
```

Comment:
Instead to transform the genres to a List<String> use it as a List<Genres>

Reply:
done